### PR TITLE
[GUI] Fix invalid MN collateral amount and align every static 10k PIV strings.

### PR DESCRIPTION
**Pull request by furszy**
Since #2296, we have a different masternode collateral output amount for regtest. As the GUI has the 10k value harcoded in some places, the masternode creation wizard got broken and there are some misaligned strings as well.

This PR fixes it and aligns every static 10k PIV string to be set dynamically consuming the chain params value.

### CODE REVIEW

File: src/qt/pivx/masternodewizarddialog.cpp [hunk 1]
```diff
@@ -50,10 +59,20 @@ MasterNodeWizardDialog::MasterNodeWizardDialog(WalletModel *model, QWidget *pare
     setCssProperty(ui->labelMessage1a, "text-main-grey");
     setCssProperty(ui->labelMessage1b, "text-main-purple");
 
+    QString collateralAmountStr = GUIUtil::formatBalance(clientModel->getMNCollateralRequiredAmount());
+    ui->labelMessage1a->setText(tr("To create a PIVX Masternode you must dedicate %1 (the unit of PIVX) to the network (however, these coins are still yours and will never leave your possession).")
+                                   .arg(collateralAmountStr) + "<br>" + "<br>" +
+                                   tr("You can deactivate the node and unlock the coins at any time."));
+
```
Comment (random-zebra) [inline src/qt/pivx/masternodewizarddialog.cpp:66]: why no formatting here?
```suggestion
    ui->labelMessage1a->setText(formatHtmlContent(
                formatParagraph(tr("To create a PIVX Masternode you must dedicate %1 (the unit of PIVX) "
                        "to the network (however, these coins are still yours and will never leave your possession).").arg(collateralAmountStr)) +
                formatParagraph(tr("You can deactivate the node and unlock the coins at any time."))));
```

**Review by random-zebra — COMMENTED** [review]
Code ACK, minus lost html-formatting in the first screen of the mn wizard.

One minor concern I have is about the change in the FAQs. 
Someone reading on a regtest wallet might think that the same applies verbatim on all networks. 
But, on the other hand, someone using the regtest probably already knows what's he doing.

**Comment by furszy** [comment]
Yep, i'm fine either way.
if the person is using regtest, then he/she at least know that there are different networks with different consensus parameters.

File: src/qt/pivx/masternodewizarddialog.cpp (hunk 1 above)
Comment (furszy) [reply src/qt/pivx/masternodewizarddialog.cpp]: No apparent reason, just added it.

**Comment by furszy** [comment]
Feedback tackled, this is a bug blocking masternodes to be created in regtest. Let's merge it.

**Review by random-zebra — APPROVED** [review]
utACK 3c5b5f3610c70f0c3ece51c9e95326649e86e321

**Comment by random-zebra** [comment]
merging...

**Pull request merged**